terraform/groups/ecs-service/locals.tf

```diff
@@ -10,7 +10,7 @@ locals {
   kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority  = 63
   lb_listener_paths          = ["/transactions/*/confirmation-statement/*", "/confirmation-statement/*", "/private/confirmation-statement/*", "/private/transactions/*/confirmation-statement/*/filings", "/confirmation-statement-api/healthcheck"]
-  healthcheck_path           = "/confirmation-statement-api/healthcheck"
+  healthcheck_path           = "/confirmation-statement-api"
```

Comment:
keep this line the same - the container will still need the healthcheck to be defined as:/confirmation-statement-api/healthcheck